.gitignore

```diff
@@ -4,7 +4,11 @@
 
 # Environment Specific #
 #####################
-/Projects/SeniorProject/devcycle-server/dataCollection/wsgi.py
+/dataCollection/wsgi.py
+/dataCollection/settings.py
+*.swp
+*.swo
+*~
 
 # CTAGS FILE #
-/Projects/SeniorProject/devcycle-server/dataCollection/tags
+/dataCollection/tags
```

Comment:
I've mostly been assuming they're wrong on things :)
I looked it up and it looks like "/" is the directory th gitignore is in